analyzer/src/funTest/assets/projects/synthetic/gomod-expected-output.yml

```diff
@@ -60,17 +60,13 @@ project:
     dependencies:
     - id: "GoMod::github.com/fatih/color:v1.7.0"
       linkage: "PROJECT_STATIC"
-    - id: "GoMod::github.com/hashicorp/go-multierror:v1.0.0"
-      linkage: "PROJECT_STATIC"
     - id: "GoMod::github.com/mattn/go-colorable:v0.1.4"
       linkage: "PROJECT_STATIC"
```

Comment:
commit-msg: Could you add a short hint why these two mentioned deps should not be in the vendor scope ?

Reply:
I don't understand the internals of go mod well enough to explain that, I just trust that the command generates the correct output, and in addition the link [1] from the commit message explains that the old command "has never given complete information in vendor mode".

Reply:
Thanks!